app/frontend/src/components/forms/submission/MySubmissionsTable.vue

```diff
@@ -172,13 +195,14 @@ export default {
       if (this.submissionList) {
         const tableRows = this.submissionList.map((s) => {
           return {
-            completedDate: this.getStatusDate(s, 'COMPLETED'),
+            // completedDate: this.getStatusDate(s, 'COMPLETED'),
             confirmationId: s.confirmationId,
+            lastEdited: this.getStatusDate(s),
             name: s.name,
             permissions: s.permissions,
             status: this.getCurrentStatus(s),
             submissionId: s.formSubmissionId,
-            submittedDate: this.getStatusDate(s, 'SUBMITTED'),
+            submittedDate: s.draft ? null : this.getStatusDate(s, 'SUBMITTED')
```

Comment:
I don't think you'll need this check. There won't be a submitted status for draft submissions.

Reply:
right this makes sense, ill  remove this in next commit.